packages/primitives/image/src/types.ts

```diff
@@ -8,7 +10,6 @@ export type TImage = {
   topRightRadius?: number,
```

Comment:
Why props regarding radius or size weren't moved to Themeable type like in Border or Background primitives?